pkg/flags/flags.go

```diff
@@ -34,6 +35,12 @@ func setLogLevel(logLevel string) {
 		log.Fatalf("invalid log-level: %s", logLevel)
 	}
 	log.SetLevel(level)
+
+	if level == log.DebugLevel {
+		flag.Set("stderrthreshold", "INFO")
+		flag.Set("logtostderr", "true")
+		flag.Set("v", "10")
```

Comment:
TIL you can call `flag.Set` _after_ `flag.Parse` and it will still work. I wasn't expecting that, but have verified that this implementation does what you want it to do.

Reply:
Honestly, it frightens me a little bit. There be dragons there =)